### ISSUE TITLE
Disable vertical scrolling when swiping on mobile

### ISSUE THREAD

**Issue by ja1shan** [SSENSE/vue-carousel#312]
Firstly, thank you for this Vue Componoent. It is working perfectly apart from one little issue I have.

When using it on a mobile device and swiping on the slider it can scroll the page up and down. Is there any way to prevent vertical scrolling on mobile when swiping left and right on the slider.

I tried to add e.preventDefault() on the onDrag function when (Math.abs(newOffsetX) > 5) with no luck.

Thanks

**Comment by quinnlangille** [comment]
Hey @ja1shan, just to confirm this is because the content of your slide is taller than the carousel? Can you send a fiddle or screen capture of this happening so we can properly diagnose?

**Comment by ja1shan** [comment]
I'm not sure but my Carousel and Slider seems to be the same size.
Here is an example https://jsfiddle.net/gvzn6mt3/5/

**Comment by ja1shan** [comment]
I only wanted to disable the scrolling when the user is swiping (through the carousel slides), not entirely.
I did try use **e.preventDefault()** earlier with no luck.

It is fine, I will look into another solution.

An example would be
https://surmon-china.github.io/vue-awesome-swiper/
When swiping the Carousel on mobile (you can't scroll up and down by swiping up and down).

**Comment by koraykupe** [comment]
I am also looking for a solution for it. There are several ways to do it. I am trying to find the most basic one.

**Comment by devolutionsOffice** [comment]
did anyone find a solutions for that?

**Comment by quinnlangille** [comment]
@koraykupe feel free to share if you end up implementing something on your end!

**Comment by koraykupe** [comment]
Sorry, I switched to another plugin.

**Comment by mateo2181** [comment]
@koraykupe I've tried to do it but I did't find a solution. Could you share what plugin you're using?

**Comment by notnotgabriel** [comment]
I'm facing the same issue here ⚠️ 
I applied the following solution to solve:
```
<carousel
    @touchstart.native="onTouchStart"
    @touchend.native="onTouchEnd"
  >
</carousel>


onTouchStart() {
  document.querySelector('body').style.overflow = 'hidden';
},
onTouchEnd() {
  document.querySelector('body').style.overflow = 'auto';
},
```

**Comment by simonmaass** [comment]
i have the same problem...
@notnotgabriel somehow this doesnt work for me... :(

**Comment by StanZhuravlev** [comment]
I have a same issue. The advise of @notnotgabriel - not working :( 

Can this issue include to roadmap?

**Comment by simonmaass** [comment]
Yeah that would be awesome - with "mobile-first" approaches in most ux discussions nowadays that would be a great feature!

**Comment by AleitaT** [comment]
Did anyone try adding `touch-action: pan-x !important;` to `.VueCarousel-slide` class? This should limit `pan-y`. Worked for me. It doesn't cure all bounce frustrations but solved this issue for me.

**Comment by mwager** [comment]
@AleitaT problem is you cannot scroll at all then :) but should be good to go for now... 

plugin support would be nice tho, similar to prop "prevent-y-scroll" from vueperslides: https://antoniandre.github.io/vueper-slides

**Comment by keelviinn** [comment]
I used "touch-pan-y" in the tailwind and it worked fine.
